slack-plugin/src/main/kotlin/slack/gradle/tasks/CheckDependencyVersionsTask.kt

```diff
@@ -37,18 +39,21 @@ public abstract class CheckDependencyVersionsTask : BaseDependencyCheckTask() {
 
   @get:Input public abstract val mappedIdentifiersToVersions: MapProperty<String, String>
 
+  // Only present for cacheability
```

Comment:
👍 It can also be useful for auditing the task and for writing tests, since it's easier to assert against file contents vs console output.